src/devdb/helpers.py

```diff
@@ -5,15 +5,169 @@
 
 BUCKET_NAME = "edm-publishing"
 
+QAQC_CHECK_SECTIONS = {
+    "Class B": "These checks are related to class A and class B unit distinctions.",
+    "Units": "These checks are related to missing or suspicious unit counts.",
+    "Duplicates": "These checks are related to identifying possible duplicates",
+    "Geography": "These checks are related to potentially missing geographic information",
+    "Other/Unique": "Miscellaneuous Checks",
+    "Outlier": "These checks are related to outlying or unexpected values",
+}
+QAQC_CHECK_DICTIONARY = {
+    "b_likely_occ_desc": {
+        "description": "Likely Class B Units: OCC Initial or Proposed contains hotel, assisted, etc. or job description contains Hotel, Motel, etc.",
+        "field_type": "boolean",
+        "section": "Class B",
+    },
+    "b_large_alt_reduction": {
+        "description": "Alterations with large (>5) reductions in Class A units",
+        "field_type": "boolean",
+        "section": "Class B",
+    },
+    "b_nonres_with_units": {
+        "description": "Non-Residential Jobs with Non-Zero Class A initial or proposed units",
+        "field_type": "boolean",
+        "section": "Class B",
+    },
+    "units_co_prop_mismatch": {
+        "description": "New Building and Alterations Jobs Where Class A proposed units is not equal to co_latest_units",
+        "field_type": "boolean",
+        "section": "Units",
+    },
+    "partially_complete": {
+        "description": "Job Status of 4. Partially Completed",
+        "field_type": "boolean",
+        "section": "Other/Unique",
+    },
+    "units_init_null": {
+        "description": "Demolitions and Alterations in Residential Jobs with Null Class A Initial Units",
+        "field_type": "boolean",
+        "section": "Units",
+    },
+    "units_prop_null": {
+        "description": "New Buildings and Alterations in Residential Jobs with Null Class A Proposed Units",
+        "field_type": "boolean",
+        "section": "Units",
+    },
+    "units_res_accessory": {
+        "description": "Work only done on nonresidential structure (Garage, Shed, etc.)",
+        "field_type": "boolean",
+        "section": "Units",
+    },
+    "outlier_demo_20plus": {
+        "description": "Demolition Jobs with larger than 19 Class A Initial Units",
+        "field_type": "boolean",
+        "section": "Outlier",
+    },
+    "outlier_nb_500plus": {
+        "description": "New Building Jobs with more than 499 Class A proposed units",
+        "field_type": "boolean",
+        "section": "Outlier",
+    },
+    "outlier_top_alt_increase": {
```

Comment:
want to note this does not make sense to include in the line chart visualization because it always just shows 20 but makes sense only in the display as list section.